test/testdrive/disabled/kafka-avro-debezium-transaction.td

```diff
@@ -9,6 +9,9 @@
 
 # Test support for Avro sources without using the Confluent Schema Registry.
 
+# This test is broken.
+# See: https://github.com/MaterializeInc/materialize/issues/13629
```

Comment:
See above, can you include a link to this test in the GH issue?

Reply:
Did you mean to move this into a different directory? For the rest of the tests that you disabled you used `skip-if`

Reply:
Yes, `test/testdrive` has a `disabled` sub-directory which seems to be the place where we move disabled .td files. Such a sub-directory doesn't exist for `test/debezium`, so I figured adding a skip-if would be less intrusive.

But checking again, it looks like there are already test files in `test/debezium/postgres` that are disabled by appending a suffix like `.gh6570`. I'll do the same for the two new failing tests.